.vscode/launch.json

```diff
@@ -99,6 +99,24 @@
       },
       "outFiles": ["${workspaceFolder}/dist/**/*.js"]
     },
+    {
+      "name": "Launch w/ empty folder",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}",
+      "args": [
+        "--extensionDevelopmentPath=${workspaceRoot}",
+        "${workspaceFolder}/test-data/empty-folder"
+      ],
+      "stopOnEntry": false,
+      "sourceMaps": true,
+      "env": {
+        "IS_DEBUG": "true",
+        "CALVA_DEV_GA": "FUBAR-69796730-4",
+        "DEBUG": "no-debug-universal-analytics"
+      },
+      "outFiles": ["${workspaceFolder}/dist/**/*.js"]
+    },
```

Comment:
This is so that we can manually test the use case of no active editor in the debugger.